components/dashboard/src/teams/TeamSettings.tsx

```diff
@@ -82,7 +121,39 @@ export default function TeamSettings() {
                 title="Settings"
                 subtitle="Manage general team settings."
             >
-                <h3>Delete Team</h3>
+                <h3>Team Name</h3>
+                <p className="text-base text-gray-500 max-w-2xl">
+                    This is your team's visible name within Gitpod. For example, the name of your company.
+                </p>
+                {errorMessage && (
+                    <Alert type="error" closable={true} className="mb-2 max-w-xl rounded-md">
+                        {errorMessage}
+                    </Alert>
+                )}
+                {updated && (
+                    <Alert type="message" closable={true} className="mb-2 max-w-xl rounded-md">
+                        Profile information has been updated.
+                    </Alert>
+                )}
+                <div className="flex flex-col lg:flex-row">
+                    <div>
+                        <div className="mt-4">
```

Comment:
**suggestion**: The amount of space between items creates relationships and hierarchy.

```suggestion
                        <div className="mt-4 mb-3">
```
---

Posting below a screenshot from **Foundations / Spacing** in the component library in our Figma team.

| Spacing Guidelines |
|-|
| ![Spacing](https://user-images.githubusercontent.com/120486/205070541-e116ccfe-83d5-41c2-bbbf-f91726934e31.png) |